src/pages/workspaces/workspace/ShareWorkspaceModal/ShareWorkspaceModal.ts

```diff
@@ -266,4 +269,31 @@ const ShareWorkspaceModal: React.FC<ShareWorkspaceModalProps> = (props: ShareWor
   );
 };
 
+const ProtectedDataWarning: React.FC = () => {
+  const msg =
+    'Do not share Unclassified Confidential Information with anyone unauthorized to access such information, ' +
+    'as it violates US Federal Policy (ie FISMA, FIPS-199, etc) ' +
+    'unless explicitly authorized by the dataset manager or governed by your own agreements';
+
+  return div(
+    {
+      role: 'textbox',
```

Comment:
I think we might want to make this a role of `alert`, as that will cause it to be announced. A role of `alert` is more commonly used for form errors or things that require immediate intervention, but this is a serious enough warning that I think it's worth using in this case too.

The other alternative I can think of is to link this block to the title of the modal or the user email input via `aria-describedby`, but that would be more work and I'm not sure it would be that much better of an experience.

Reply:
Never mind, I tried changing it to `alert` and it didn't get read out… probably because it's always in the HTML, just not visible until the modal is displayed.

Let me play around with `aria-describedby`.

Reply:
If I link the warning text to the user email input via `aria-describedby`, it gets read out as additional content… if the user chooses to read out the additional content.

I think we can punt on linking the warning text to a control since the user likely would read though the content in the dialog (and thus the warning) before filling out the fields.